rpc/types/Cargo.toml

```diff
@@ -9,14 +9,16 @@ repository  = "https://github.com/Cuprate/cuprate/tree/main/rpc/types"
 keywords    = ["cuprate", "rpc", "types", "monero"]
 
 [features]
-default = []
+default = ["serde", "epee"]
+serde   = ["dep:serde"]
+epee    = ["dep:cuprate-epee-encoding"]
```

Comment:
All types enabled by default, only `serde` and `epee` are feature gated (enabled by default though).